skyrim-platform/src/platform_se/skyrim_platform/GameEventSinks.cpp

```diff
@@ -629,3 +630,33 @@ RE::BSEventNotifyControl GameEventSinks::ProcessEvent(
 
   return RE::BSEventNotifyControl::kContinue;
 };
+
+RE::BSEventNotifyControl GameEventSinks::ProcessEvent(
+  const RE::TESSpellCastEvent* event,
+  RE::BSTEventSource<RE::TESSpellCastEvent>* eventSource)
+{
+  auto converted =
```

Comment:
casting to `PapyrusSendEvent::TESSpellCastEvent` isn't required here. we should use `RE::TESSpellCastEvent`

Reply:
`RE::TESSpellCastEvent` declared in RE, but the struct inside is not define like in other finished on. For example, I can include like for `RE::TESHitEvent`, cause there is no header for `TESSpellCastEvent`.
So that why i'm using cast here.